docs/src/catalyst_functionality/dsl_description.md

```diff
@@ -86,24 +86,24 @@ Numeric parameter values do not need to be set when the model is created, i.e.
 Catalyst supports symbolic parameters too:
 ```@example tut2
 rn = @reaction_network begin
-  p, ∅ --> X
-  d, X --> ∅
+  k1, X --> Y
+  k2, Y --> X
 end
 ```
 All symbols that do not appear as a substrate or product in a reaction are
 designated by Catalyst as a parameter (i.e. all symbols appearing only within
-rate expressions and/or as stoichiometric coefficients). In this example `X`
-appears as a substrate and product, but not `p` or `d`. Hence `p` and `d` are
-designated as parameters. It is possible to manually specify what should be
-considered a species or parameter. This is described later in the tutorial.
+rate expressions and/or as [stoichiometric coefficients](@ref parametric_stoichiometry)). In this example `X` and `Y`
+appears as a substrates and products, but neither `k1` nor `k2`. Hence `k1` and `k2` are
+designated as parameters. Later in this tutorial, we will describe how to manually specify what should be
+considered a species or parameter.
 
-## Production, Destruction and Stoichiometry
+## Production, Destruction, and Stoichiometry
 Sometimes reactants are produced/destroyed from/to nothing. This can be
-designated using either `0` or `∅`:
+designated using `0` (`∅` can also be used as an alternative to `0`):
```

Comment:
I prefer the old wording here, why single out one notation?

Reply:
I thought maybe we wanted to go for the less uni-code stuff. But I like the use of `∅` so will revert.